docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc

```diff
@@ -0,0 +1,395 @@
+----
++
+[source,sh,subs="attributes"]
+----
+NAME            HEALTH   AVAILABLE   EXPECTED   VERSION      AGE
+elastic-agent   green    3           3          {version}    14s
+fleet-server    green    1           1          {version}    19s
+
+----
+
+. List all the Pods belonging to a given Elastic Agent specification.
++
+[source,sh]
+----
+kubectl get pods --selector='agent.k8s.elastic.co/name=elastic-agent'
+----
++
+[source,sh]
+----
+NAME                        READY   STATUS    RESTARTS   AGE
+elastic-agent-agent-t49fd   1/1     Running   0          54s
+elastic-agent-agent-xbcxr   1/1     Running   0          54s
+elastic-agent-agent-zqp55   1/1     Running   0          54s
+----
+
+. Access logs for one of the Pods.
++
+[source,sh]
+----
+kubectl logs -f elastic-agent-agent-xbcxr
+----
+
+. Configure policy used by Elastic Agents.
++
+See Fleet link:https://www.elastic.co/guide/en/fleet/current/agent-policy.html[docs] for details.
+
+[id="{p}-elastic-agent-fleet-configuration"]
+== Configuration
+
+experimental[]
+
+Fleet-managed Elastic Agents must connect to Fleet Server to receive their configurations. You can deploy Fleet Server instances using ECKs Agent CRD with appropriate configuration, as below. To learn more about Fleet architecture and components involved please refer to Fleet link:https://www.elastic.co/guide/en/fleet/current/fleet-server.html[documentation].
+
+[id="{p}-elastic-agent-fleet-configuration-fleet-mode-and-fleet-server"]
+=== Fleet mode and Fleet Server
+Running both Fleet Server and Elastic Agent in Fleet-managed mode requires setting `mode` configuration element to `fleet`.
+
+[source,yaml,subs="attributes,+macros"]
+----
+apiVersion: agent.k8s.elastic.co/v1alpha1
+kind: Agent
+metadata:
+  name: quickstart
+spec:
+  mode: fleet
+----
+
+Running Fleet Server requires setting `fleetServerEnabled` configuration element to `true`, as below. As `false` is the default value, it can be left unset for any other case.
+
+[source,yaml,subs="attributes,+macros"]
+----
+apiVersion: agent.k8s.elastic.co/v1alpha1
+kind: Agent
+metadata:
+  name: quickstart
+spec:
+  mode: fleet
+  fleetServerEnabled: true
+----
+
+[id="{p}-elastic-agent-fleet-configuration-required-kibana-configuration"]
+=== Required Kibana configuration
+
+There are two settings that must be set correctly in Kibana configuration for Fleet to work.
+
+[source,yaml,subs="attributes,+macros"]
+----
+apiVersion: kibana.k8s.elastic.co/v1
+kind: Kibana
+metadata:
+  name: kibana
+spec:
+  config:
+    xpack.fleet.agents.elasticsearch.host: "https://elasticsearch-es-http.default.svc:9200"
+    xpack.fleet.agents.fleet_server.hosts: ["https://fleet-server-agent-http.default.svc:8220"]
+----
+
+`xpack.fleet.agents.elasticsearch.host` has to point to Elasticsearch cluster that Elastic Agents should send the data to. For Elasticsearch clusters managed by ECK, ECK creates a Service with `https://${ES_RESOURCE_NAME}-es-http.${ES_RESOURCE_NAMESPACE}.svc:9200` url.
```

Comment:
I understand what you mean here, however I'm wondering if some users may think that `${ES_RESOURCE_NAME}` or `${ES_RESOURCE_NAMESPACE}` are valid env. variables generated automatically by ECK.